account_move_template/models/account_move_template.py

```diff
@@ -182,7 +182,7 @@ class AccountMoveTemplateLine(models.Model):
     @api.depends("is_refund", "account_id", "tax_line_id")
     def _compute_tax_repartition_line_id(self):
         for record in self.filtered(lambda x: x.account_id and x.tax_line_id):
-            tax_repartition = "refund_tax_id" if record.is_refund else "invoice_tax_id"
+            tax_repartition = "tax_id"
```

Comment:
Why removing the refund part?

Reply:
I removed it as both the invoice_tax_id and refund_tax_id fields were merged into the tax_id field, as seen in the following PR: https://github.com/odoo/odoo/commit/e41f2f720df2d1f653d4f715fbefdbe49f9c6574#diff-1ee3fce434db0c4e897973eebf5c1be501196cbd413e6c250ecc973238f939b9L1247